internal/slicer/slicer_test.go

```diff
@@ -1737,6 +1737,94 @@ var slicerTests = []slicerTest{{
 		"/dir/file2": "file 0644 28121945 <2> {test-package2_myslice}",
 		"/hardlink2": "file 0644 28121945 <2> {test-package2_myslice}",
 	},
+}, {
+	summary: "Mutations for hard links are forbidden",
+	slices: []setup.SliceKey{
+		{"test-package", "myslice"}},
+	pkgs: []*testutil.TestPackage{{
+		Name: "test-package",
+		Data: testutil.MustMakeDeb([]testutil.TarEntry{
+			testutil.Dir(0755, "./"),
+			testutil.Dir(0755, "./dir/"),
```

Comment:
Can we remove `./dir/`, I think it is only cluttering the output and not adding anything to these tests.